src/networkfilemanager.cpp

```diff
@@ -1683,7 +1700,8 @@ PROJ_NETWORK_HANDLE *CurlFileHandle::open(PJ_CONTEXT *ctx, const char *url,
         return nullptr;
 
     auto file =
-        std::unique_ptr<CurlFileHandle>(new CurlFileHandle(url, hCurlHandle));
+        std::unique_ptr<CurlFileHandle>(new CurlFileHandle(
+            url, hCurlHandle, ctx->ca_bundle_path.c_str()));
```

Comment:
The original code will always pass a non-null pointer to CurlFileHandle constructor. So this should probably be:
```suggestion
            url, hCurlHandle, ctx->ca_bundle_path.empty() ? nullptr : ctx->ca_bundle_path.c_str()));
```

Reply:
Thanks :+1: